src/python/pants/bin/remote_pants_runner.py

```diff
@@ -86,6 +89,44 @@ def _setup_logging(self):
     root.setLevel(log_level)
     root.addHandler(handler)
 
+  @staticmethod
+  def _backoff(attempt):
+    """Minimal backoff strategy for daemon restarts."""
+    time.sleep(attempt + (attempt - 1))
```

Comment:
Just wondering if there is any particular reason to use `attempt + (attempt - 1)`?

Reply:
just trying to keep the retry overhead to <10 seconds and this staggers nicely as 1, 3, 5 seconds for 3 retries. the backoff curve here needn't be as sharp as e.g. a distributed system where things like thundering herd are a thing.